src/get-started/design-overview.md

```diff
@@ -224,62 +225,67 @@ which more than 1/3 (but less than 2/3) of the validators are compromised.
-  already stored items
-- [`ProofMapIndex`](../architecture/storage.md#proofmapindex)
-  extends the
+  concatenated children data. `ProofListIndex` only allows to append the data
+  or update the already stored items
+- [`SparseListIndex`][sparse-list-index] represents a `ListIndex` that may
+  contain "gaps". It provides the possibility to delete items not only from
+  the end of the list, but from any part thereof. Such deletions do not break
+  the order of the indices inside the list
+- [`ProofMapIndex`](../architecture/merkledb.md#proofmapindex) extends the
   map. It is based on a Merkle Patricia tree, implemented as a binary tree.
-  Leaves of the tree keep the actual
-  values from the map. Intermediate nodes consist of the following four parts:
+  Leaves of the tree keep the actual values from the map. Intermediate nodes
+  consist of the following four parts:
 
     - Hash of the left child value
     - Hash of the right child value
     - Key for the left child node
     - Key for the right child node
 
 - `ValueSetIndex` and `KeySetIndex` both implement sets, and both
-  reduce them to
-  maps (as it is commonly done in programming languages). `ValueSetIndex`
-  maps hashes of the set items to the items themselves,
+  reduce them to maps (as it is commonly done in programming languages).
+  `ValueSetIndex` maps hashes of the set items to the items themselves,
   while `KeySetIndex` maps set items to `null`.
   Thus, `KeySetIndex` is preferable when set items have relatively
   short serialization and need to be iterated in a deterministic order.
   `ValueSetIndex` is a better match for complex items or items that need to be
   looked up by a hash.
 
-Both `ListIndex` and `ProofListIndex` support updating by index and
-appending only. `MapIndex` and `ProofMapIndex` allow inserting,
-updating or deleting key-value pairs. `KeySetIndex` and `ValueSetIndex` support
-adding and removing elements from the set. Finally, all collections support
-iterations over items (or keys, values, and key-value pairs in the case of
-  maps).
+  Both `ListIndex` and `ProofListIndex` support updating by index and
+  appending only. `MapIndex` and `ProofMapIndex` allow inserting,
+  updating or deleting key-value pairs. `KeySetIndex` and `ValueSetIndex`
+  support adding and removing elements from the set. Finally, all collections
+  support iterations over items (or keys, values, and key-value pairs in the
+  case of maps).
+- `Entry` represents a set that contains only one item.
 
 ### Proofs
 
 `ProofListIndex` and `ProofMapIndex` allow efficiently
 creating a proof that specific values are saved under particular keys.
 To prove that, it is sufficient to return a list of hashes from
 the tree root to a particular cell (a Merkle path). Merkle Patricia
-tables also allow to generate proofs that there is no data in the
+stores also allow to generate proofs that there is no data in the
```

Comment:
IMO, that sounds somewhat strange — like a place with a fancy name (do we use a 'store' elsewhere?)